lib/repository_pulling_worker.rb

```diff
@@ -0,0 +1,12 @@
+class RepositoryPullingWorker
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  sidekiq_options retry: false, queue: 'default'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.